config.go

```diff
@@ -130,7 +130,10 @@ func (element *ConfigST) GetWebRTCPortMax() uint16 {
 
 func loadConfig() *ConfigST {
 	var tmp ConfigST
-	data, err := ioutil.ReadFile("config.json")
+	data, err := ioutil.ReadFile("/tmp/config.json")
```

Comment:
maybe it's better to use some argument/environment variable and fall back to default (`"config.json"`) if it's not set